packages/@guardian/source-foundations/src/index.ts

```diff
@@ -1 +1,2 @@
-export { transitions } from '../../../../src/core/foundations/src/animation';
+// animation
+export { transitions } from './animation';
```

Comment:
Just to jog my memory, did we decide to flatten foundations in v4 (i.e only export from the top level) or are we going to maintain the existing import structure for now?

Reply:
discussed offline – flatten and rename export names where they clash, then see how many that is